src/main/java/seedu/duke/commands/DeleteCommand.java

```diff
@@ -23,27 +27,34 @@ public int getTaskNumber() {
         return taskNumber;
     }
 
-    public DeleteCommand(String moduleCode) {
-        this.moduleCode = moduleCode;
+    public String getTaskModule() {
+        return taskModule;
     }
 
-    public DeleteCommand(int taskNumber) {
-        this.taskNumber = taskNumber;
+    public DeleteCommand(String moduleCode) {
+        this.moduleCode = moduleCode;
     }
```

Comment:
Just curious, is the moduleCode here the same to the moduleName in the AddCommand.java?